src/Microsoft.HttpRepl/Commands/SetHeaderCommand.cs

```diff
@@ -18,14 +20,22 @@ namespace Microsoft.HttpRepl.Commands
 {
     public class SetHeaderCommand : ICommand<HttpState, ICoreParseResult>
     {
-        private static readonly string Name = "set";
+        private static readonly string CommandName = "set";
         private static readonly string SubCommand = "header";
 
+        private readonly ITelemetry _telemetry;
+
+        public string Name => "SetHeader";
```

Comment:
Is this the only command using PascalCase for the Name property?  all the others have been lowercase.

Reply:
There was one other - I changed them both to camel for consistency